src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -945,6 +945,22 @@ public string PlaceholderText
             }
         }
 
+        /// <summary>
+        /// The Selected Text property.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done, Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string SelectedText
+        {
+            get
+            {
+                string temp;
+                GetProperty(TextEditor.Property.SELECTED_TEXT).Get(out temp);
```

Comment:
could you resolve the build warning shown below?

`src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs#L959
Call System.IDisposable.Dispose on object created by 'GetProperty(TextEditor.Property.SELECTED_TEXT)' before all reference...`

basically, there must be NO warning.

Reply:
hmmm, What .Net SDK are you using? Also, are you doing adding additional compilation flags?
I am asking because I could not see this warning when building the project